database/dbtest/token.go

```diff
@@ -0,0 +1,29 @@
+package dbtest
+
+import "github.com/coreos/clair/pkg/pagination"
+
+type MockPage struct {
```

Comment:
This is the implementation I'm dubious about.
https://github.com/coreos/clair/blob/master/database/pgsql/notification.go#L214
Here notification.go expects the Page struct to be exactly this, however, this is an implicit requirement.
If any implementation uses any other Page struct implementation, we'll have trouble.